packages/gridstack/src/components/SaveDashboardModal/SaveDashboardModal.vue

```diff
@@ -11,6 +11,8 @@
       name="name"
       class="base-input"
     />
+    <ErrorBox v-if="showError" class="error-message" :error="nameError" />
```

Comment:
The ErrorBox will auto hide if no error given. So if you pass an error instead of the `showError` bool you can remove the v-if here. The documentation has not stated that. I added additional docs here https://github.com/lion5/component-library/pull/202 . So u can check if it's clear enough now.

Reply:
Changed to passing the error